README.md

```diff
@@ -177,6 +179,12 @@ Rspack and Rsbuild support most of the [unplugin](https://github.com/unplugin),
 
 - [@rsbuild/plugin-solid](https://rsbuild.dev/plugins/list/plugin-solid): Provides support for Solid.
 
+#### For Electron
+
+- [@electron-rsbuild/plugin-main](https://electron-rsbuild.org/plugins/list/plugin-main): Provides support for Electron main process.
+- [@electron-rsbuild/plugin-preload](https://electron-rsbuild.org/plugins/list/plugin-preload): Provides support Electron preload process.
+- [@electron-rsbuild/plugin-renderer](https://electron-rsbuild.org/plugins/list/plugin-renderer): Provides support Electron renderer process.
+
```

Comment:
electron-rsbuild has been added to `Ecosystem`, so there is no need to add it to other sections.

```suggestion
```